client/actions/events.js

```diff
@@ -613,6 +613,16 @@ const closeAdvancedSearch = () => (
 )
 
 /**
+ * Opens the Event in preview/read-only mode
+ * @param {object} event - The Event ID to preview
+ * @return Promise
+ */
+const previewEvent = (event) => ({
+    type: EVENTS.ACTIONS.PREVIEW_EVENT,
+    payload: get(event, '_id', event || true),
```

Comment:
I find a bit confusing to mix `_id` and boolean here.
Can a user preview an event that has no `_id` (not saved event) ?